app/components/travis-form.js

```diff
@@ -1,10 +1,10 @@
 import Component from '@ember/component';
-import { computed } from '@ember/object';
 import { gt, filterBy, not } from '@ember/object/computed';
 import * as validators from 'travis/utils/form-validators';
 
 export default Component.extend({
   tagName: '',
+  className: 'travis--travis-form',
```

Comment:
We usually strive to tie component CSS class to component name where possible. So would it be possible to rename the `className` to just `travis-form`. Also, it's not a strict rule, but we sometimes use the [BEM naming convention](http://getbem.com/naming/) for CSS styles, so this name will conflict with it (`--travis-form` does not look like a valid modifier).

And, as far as I understand, there's a typo: `className` should be [`classNames`](https://api.emberjs.com/ember/3.10/classes/Component/properties/classNames?anchor=classNames)

Reply:
Do we have any doc about our code / style rules?

Reply:
No, unfortunately, we don't have such document currently. That's why I mentioned that it's not a strict rule, but like a general recommendation, because it may cause confusion with other developers.